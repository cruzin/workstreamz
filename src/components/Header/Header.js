import React from 'react';
import PropTypes from 'prop-types';

import './Header.scss';

const Header = ({ children }) => (
  <header className='header'>
    <div>
      <h1>Bilyeli</h1>
      <p>Boilerplate for React/Redux app.</p>
    </div>
    <div>{children}</div>
  </header>
);

Header.propTypes = {
  children: PropTypes.node,
};

export default Header;
