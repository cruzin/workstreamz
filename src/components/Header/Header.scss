.header {
  width: 100%;
  padding: 50px 0;
  margin-bottom: 30px;
  background: #fafbfc;
  border-bottom: 1px solid #e1e4e8;
  text-align: center;
}
