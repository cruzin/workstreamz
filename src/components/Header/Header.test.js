/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Header from './Header';

describe('Header', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header />, div);
  });

  it('renders markup correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
