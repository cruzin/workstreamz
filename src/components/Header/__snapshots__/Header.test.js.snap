// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders markup correctly 1`] = `
<header
  className="header"
>
  <div>
    <h1>
      Bilyeli
    </h1>
    <p>
      Boilerplate for React/Redux app.
    </p>
  </div>
  <div />
</header>
`;
